opencti-platform/opencti-graphql/src/graphql/telemetryPlugin.js

```diff
@@ -36,7 +36,7 @@ export default {
         };
 
         if (!sendContext.operationName) {
-          logApp.error('TELEMETRY PLUGIN UNDEFINED OPERATION', { query: stripIgnoredCharacters(sendContext.request?.query ?? 'undefined') });
+          logApp.info('[TELEMETRY] GraphQL operation is not named', { query: stripIgnoredCharacters(sendContext.request?.query ?? 'undefined') });
```

Comment:
we can use "unnamed" instead of "not named", it is more formal and standard for technical documentation